src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/GeneratorTests.Baselines.cs

```diff
@@ -15,6 +15,107 @@ public partial class ConfigurationBindingGeneratorTests
+                            }
+
+                            public class MyClass
+                            {
+                                public string MyString { get; set; }
+                                public int MyInt { get; set; }
+                                public List<int> MyList { get; set; }
+                                public Dictionary<string, string> MyDictionary { get; set; }
+                            }
+                        }
+                    """;
+
+            var (d, r) = await RunGenerator(source);
+            Assert.Equal(1, r.Length);
+            Assert.Empty(d);
+            Assert.Equal(151, r[0].SourceText.Lines.Count); // No need to check line by line, other tests already doing that. 
+        }
+
+        [Fact]
+        public async Task Get_TypeOf_NamedParameters()
+        {
+            string source = """
+                        using System.Collections.Generic;
+                        using Microsoft.Extensions.Configuration;
+
+                        public class Program
+                        {
+                            public static void Main()
+                            {
+                                ConfigurationBuilder configurationBuilder = new();
+                                IConfigurationRoot config = configurationBuilder.Build();
+
+                                MyClass configObj = new();
+                                var obj = ConfigurationBinder.Get(type: typeof(MyClass), configuration: config);
+                            }
+
+                            public class MyClass
+                            {
+                                public string MyString { get; set; }
+                                public int MyInt { get; set; }
+                                public List<int> MyList { get; set; }
+                                public Dictionary<string, string> MyDictionary { get; set; }
+                            }
+                        }
+                    """;
+
+            var (d, r) = await RunGenerator(source);
+            Assert.Equal(1, r.Length);
+            Assert.Empty(d);
+            Assert.Equal(188, r[0].SourceText.Lines.Count);
+        }
+
+        [Fact]
+        public async Task GetValue_NamedParameters()
+        {
+            string source = """
+                        using System.Collections.Generic;
+                        using Microsoft.Extensions.Configuration;
+
+                        public class Program
+                        {
+                            public static void Main()
+                            {
+                                ConfigurationBuilder configurationBuilder = new();
+                                IConfigurationRoot config = configurationBuilder.Build();
+
+                                var str = ConfigurationBinder.GetValue(key: "key", configuration: config, type: typeof(string));
```

Comment:
We also need variants for regular extension method call syntax - `config.GetValue(key: "key", type: typeof(string))`. Here & wherever applicable.

Reply:
Done